core/src/createTextMaskInputElement.js

```diff
@@ -25,160 +75,135 @@ export default function createTextMaskInputElement({
     providedMask = providedMask.mask
   }
```

Comment:
This code now gets executed at every `update` call. It used to be executed during component creation only. Can we retain this behavior? Same goes for lines 84 to 86.